pkg/model/awsmodel/autoscalinggroup.go

```diff
@@ -52,6 +55,13 @@ func (b *AutoscalingGroupModelBuilder) Build(c *fi.ModelBuilderContext) error {
 	for _, ig := range b.InstanceGroups {
 		name := b.AutoscalingGroupName(ig)
 
+		if featureflag.SpotinstHybrid.Enabled() {
```

Comment:
Does this mean when I activate this feature flag I just override the use of AutoScaling Groups? 
I am not a big fan of changing behaviour with feature flags. As the name suggests, it should add features, not modify.

Reply:
No, it's not. It just means that you'd like to use Spotinst for one or more instance groups. It allows users to continue managing their instance groups through AWS Auto Scaling groups, except for specific ones labeled with our metadata label (`spotinst.io/hybrid: "true"`). So, if you toggle the feature flag on but without setting the metadata label you do not override anything.

Reply:
Makes sense, thank you for clarifying!